docs/apple-privacy-manifest.md

```diff
@@ -0,0 +1,259 @@
+
+# Example
+Let's look at how you would add a Privacy Manifest file to an application that uses the following API's:
+
+* [NSUserDefaults](https://learn.microsoft.com/dotnet/api/foundation.nsuserdefaults) 
+* [NSProcessInfo.SystemUptime](https://learn.microsoft.com/dotnet/api/foundation.nsprocessinfo.systemuptime)
+* [NSFileManager.ModificationDate](https://learn.microsoft.com/dotnet/api/foundation.nsfilemanager.modificationdate)
+
+How they are used is not that important for this example, but the `why` will determine the reason code needed for the privacy manifest.
+## Creating the `PrivacyInfo.xcprivacy` file
+
+We will start by building the contents of the `PrivacyInfo.xcprivacy` file, and then go through each supported platform and how to properly configure your project so the file in included in the bundle properly.
+
+Since the application is based on .NET, a privacy manifest is required. Let's walkthrough the steps needed to add a privacy manifest that declares our usages of the above three API's.
+
+1. Open Xcode and either create new `App` project, or open an existing one.
+1. In Xcode use the File->New->File menu to open the `Choose a new template for your file:` dialog box.
+1. Scroll down until you find the `App Privacy` template.
+1. Select the `App Privacy` template and then click `Next`
+1. In the `Save As` dialog, leave the filename as `PrivacyInfo` as that is the required name for the file.
+1. Click `Create` and close Xcode.
+1. Use Finder to copy the `PrivacyInfo.xcprivacy` file from the Xcode project to your documents folder for now.
+1. The Xcode project is no longer needed and can be deleted.
+
+You should now have a file named `PrivacyInfo.xcprivacy` under your documents folder with contents similar to:
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<!--
+... omitted for brevity
+-->
+<plist version="1.0">
+<dict/>
+</plist>
+```
+Use your favorite text editor, like [Visual Studio Code](https://aka.ms/vscode), to open the file for editing.
+
+You can add the entries for the API's usage to the `PrivacyInfo.xcproject` as follows: 
+
+1. Edit the `PrivacyInfo.xcprivacy` file to appear as follows:
+    ```xml
+    <?xml version="1.0" encoding="UTF-8"?>
+    <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+    <!--
+    ... omitted for brevity
+    -->
+    <plist version="1.0">
+    <dict>
+        <key>NSPrivacyAccessedAPITypes</key>
+        <array>
+        </array>
+    </dict>
+    </plist>
+    ```
+    This adds the `NSPrivacyAccessAPITypes` key where each category usage will be added.
+1. Since the .NET runtime and BCL include API's from the [File timestamp][FileTimestampAPIs], [System boot time][SystemBootTimeAPIs], [Disk space][DiskSpaceAPIs] API categories, add the following to the `NSPrivacyAccessedAPITypes` array:
```

Comment:
```suggestion
1. Since the .NET runtime and BCL include APIs from the [File timestamp][FileTimestampAPIs], [System boot time][SystemBootTimeAPIs], [Disk space][DiskSpaceAPIs] API categories, add the following to the `NSPrivacyAccessedAPITypes` array:
```